src/main/java/com/commercetools/sync/sdk2/commons/utils/AssetsUpdateActionUtils.java

```diff
@@ -1,22 +1,21 @@
 package com.commercetools.sync.sdk2.commons.utils;
 
-import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
 import static com.commercetools.sync.commons.utils.OptionalUtils.filterEmptyOptionals;
+import static com.commercetools.sync.sdk2.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
 import static java.lang.String.format;
 import static java.util.Collections.singletonList;
 import static java.util.Optional.ofNullable;
 import static java.util.stream.Collectors.*;
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
 
-import com.commercetools.sync.commons.BaseSyncOptions;
+import com.commercetools.api.models.ResourceUpdateAction;
+import com.commercetools.api.models.common.Asset;
+import com.commercetools.api.models.common.AssetDraft;
 import com.commercetools.sync.commons.exceptions.BuildUpdateActionException;
```

Comment:
I modified them right away since those classes in old package needs no migration actually.
https://github.com/commercetools/commercetools-sync-java/pull/872/commits/4feeabfcd6f91e105614fa8a38fbf136573cb6f0